vspec.py

```diff
@@ -225,15 +244,31 @@ def yaml_compose_node(parent, index):
         except yaml.parser.ParserError as e:
             raise VSpecError(file_name, line + 1, e)
 
-        node.__line__ = line + 1
-        node.__file_name__ = file_name
+        if node.value == '$include$':
+            node.value = f'$include${load_flat_model.include_index}'
```

Comment:
@magnusfeuer Is this **f** in ```f'$include...'``` special syntax or a typo?
Python seems to complain about it:
```
    node.value = f'$include${load_flat_model.include_index}'
                                                           ^
SyntaxError: invalid syntax
```
If it's some python3 unique thing, it doesn't make sense since the top of the file specifies python2 as interpreter...

Reply:
@gunnarx  I am confused, where does it mention py2? The setup.py explicitly mentions py 3, and so does vspec2json.py.  vspec.py has no shebang at all (and if I understand correctly isn't called directly anyway), or am I missing something?

Reply:
I'm so sorry, it was my own program that was requiring python2 and I was totally confused apparently.  Thanks @SebastianSchildt for checking my BS...  So we can conclude that the syntax is python3 supported.  I'll fix my program to be python3 compliant and move on.